### PR TITLE
Network Monitor

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Update Network Monitor doc

### Motivation
Audit of Monitor section

### Preview link
https://docs-staging.datadoghq.com/ruth/monitors6/monitors/monitor_types/network

### Additional Notes
Do not merge until monitor team reviews

### CODE REVIEW

File: content/en/monitors/monitor_types/network.md [hunk 1]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
 
-4. Configure **notification options**. Refer to the [Notifications][3] documentation for a detailed walkthrough of the common notification options.
+### Network Status
 
-## Network Metric
+#### Pick a check
 
-1. Choose a **network metric**. You are able to choose either the `TCP` or `HTTP` response time metric.
+* Choose a network check type (`ssl`, `http`, or `tcp`).
+* Choose a specific check or `All monitored <TYPE> endpoints`.
```
Comment (unclebconnor) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
* Choose a specific endpoint or `All monitored <TYPE> endpoints`.
```

File: content/en/monitors/monitor_types/network.md [hunk 2]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
+
+Set up the check alert:
+
+1. Trigger a separate alert for each `<GROUP>` reporting your check.
+
+    Check grouping is specified either from a list of known groupings or by you. For network monitors, the per-check grouping is explicitly known. For example, the HTTP check is tagged with `host`, `instance`, and `url`.
+
+2. Trigger the alert after selected consecutive failures: `<NUMBER>`
+
+    Each check run submits a single status of `OK`, `WARN`, `CRITICAL`, or `UNKNOWN`. Choose how many consecutive runs with the `CRITICAL` status trigger a notification. For example, your HTTP check might have a single blip where connection fails. If you set this value to `> 1`, the blip is ignored but a problem with more than one consecutive failure triggers a notification.
```
Comment (unclebconnor) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
    Each check run submits a single status of `OK`, `WARN`, or `CRITICAL`. Choose how many consecutive runs with the `CRITICAL` status trigger a notification. For example, your HTTP check might have a single blip where connection fails. If you set this value to `> 1`, the blip is ignored but a problem with more than one consecutive failure triggers a notification.
```

No `UNKNOWN` statuses for any of these checks

File: content/en/monitors/monitor_types/network.md [hunk 3]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
+
+See the [metric monitor][4] documentation for information on [No data][5], [Auto resolve][6], and [Evaluation delay][7] options.
+
+#### Notifications
+
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][8] page.
+
+### Network Metric
+
+Create a network metric monitor by following the instructions in the [metric monitor][8] documentation. Using the network metric monitor type ensures the monitor can be selected by the integration monitor type facet on the [Manage Monitors][9] page.
```
Comment (unclebconnor) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
Create a network metric monitor by following the instructions in the [metric monitor][8] documentation. Using the network metric monitor type ensures the monitor can be selected by the network monitor type facet on the [Manage Monitors][9] page.
```

File: content/en/monitors/monitor_types/network.md [hunk 4]
```diff
@@ -11,37 +11,87 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
  text: "Schedule downtime to mute a monitor."
```

File: content/en/monitors/monitor_types/network.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Network monitor
+title: Network Monitor
 kind: documentation
 description: "Check the status of TCP/HTTP endpoints"
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
description: "Check the status of TCP/HTTP endpoints."
```

File: content/en/monitors/monitor_types/network.md [hunk 6]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
 
-1. Choose a **network metric**. You are able to choose either the `TCP` or `HTTP` response time metric.
+* Choose a network check type (`ssl`, `http`, or `tcp`).
+* Choose a specific endpoint or `All monitored <TYPE> endpoints`.
 
-2. Pick the **monitor scope**. Only hosts or tags from the metric you selected are available.
+#### Pick monitor scope
 
-3. Select **alerting options**. Refer to the alert-conditions section in the [metric monitor][2] documentation for details.
+Select the scope to monitor by choosing host names, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
Select the scope to monitor by choosing hostnames, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```

File: content/en/monitors/monitor_types/network.md [hunk 7]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
 
-4. Configure **notification options**. Refer to the [Notifications][3] documentation for a detailed walkthrough of the common notification options.
+* The include field uses `AND` logic. All listed host names and tags must be present on a host for it to be included.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
* The include field uses `AND` logic. All listed hostnames and tags must be present on a host for it to be included.
```

File: content/en/monitors/monitor_types/network.md [hunk 8]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
+* The exclude field uses `OR` logic. Any host with a listed host name or tag is excluded.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
* The exclude field uses `OR` logic. Any host with a listed hostname or tag is excluded.
```

File: content/en/monitors/monitor_types/network.md [hunk 9]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
+
+Set up the check alert:
+
+1. Trigger a separate alert for each `<GROUP>` reporting your check.
+
+    Check grouping is specified either from a list of known groupings or by you. For network monitors, the per-check grouping is explicitly known. For example, the HTTP check is tagged with `host`, `instance`, and `url`.
+
+2. Trigger the alert after selected consecutive failures: `<NUMBER>`
+
+    Each check run submits a single status of `OK`, `WARN`, or `CRITICAL`. Choose how many consecutive runs with the `CRITICAL` status trigger a notification. For example, your HTTP check might have a single blip where connection fails. If you set this value to `> 1`, the blip is ignored but a problem with more than one consecutive failure triggers a notification.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
    Each check run submits a single status of `OK`, `WARN`, or `CRITICAL`. Choose how many consecutive runs with the `CRITICAL` status trigger a notification. For example, your HTTP check might have a single blip where the connection fails. If you set this value to `> 1`, the blip is ignored, but a problem with more than one consecutive failure triggers a notification.
```

File: content/en/monitors/monitor_types/network.md [hunk 10]
```diff
@@ -11,37 +11,87 @@ further_reading:
...
+    Choose how many consecutive runs with the `OK` status resolve the alert.
+
+{{% /tab %}}
+{{% tab "Cluster Alert" %}}
+
+A cluster alert calculates the percent of checks in a given status and compares it to your thresholds.
+
+Set up a cluster alert:
+
+1. Decide whether or not to group your checks according to a tag. `Ungrouped` calculates the status percentage across all sources. `Grouped` calculates the status percentage on a per group basis.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/network.md]:
```suggestion
1. Decide whether or not to group your checks according to a tag. `Ungrouped` calculates the status percentage across all sources. `Grouped` calculates the status percentage on a per-group basis.
```

**Pull request merged**